option optimize_for = LITE_RUNTIME;
import "PlayerInfo.proto";
message QiecuoStartServerRequest
{
 required PlayerInfo peerinfo = 1;
}
